packages/react-components/react-keytips-preview/library/src/index.ts

```diff
@@ -0,0 +1,9 @@
+export {
+  Keytips,
```

Comment:
adding implementation logic should not be part of project scaffolding PR. please undo this

Reply:
I meant including whole Keytips component boilerplate that was generated